### PR TITLE
Add s390x target to Makefile

### PR DESCRIPTION
**Pull request by barthy1**
This will add s390x option for cross compilation builds

Signed-off-by: Yulia Gaponenko <yulia.gaponenko1@de.ibm.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ vendor:
 	@go mod vendor
 
 .PHONY: cross
-cross: amd64 386 arm arm64 ## build cross platform binaries
+cross: amd64 386 arm arm64 s390x ## build cross platform binaries
```
Comment (sthaha) [inline Makefile]: Could you please update the README.md as well?

Comment (barthy1) [reply Makefile]: @sthaha I don't see how I can update README.md here, as Source install works as is, and so far there is no s390x build to add to tarball, deb or rpm sections.

**Comment by barthy1** [comment]
@vdemeester I've added s390x to sample .goreleaser.yml, deb, rpm and tarball are generated with correct tkn binary

**Pull request merged**